<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
    <meta name="description" content="" />
    <meta name="author" content="" />
    <title>Resume - Start Bootstrap Theme</title>
    <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
    <!-- Font Awesome icons (free version)-->
    <script src="https://use.fontawesome.com/releases/v6.3.0/js/all.js" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">

    <!-- Google fonts-->
    <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet"
        type="text/css" />
    <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
    <!-- Core theme CSS (includes Bootstrap)-->
    <link href="css/styles.css" rel="stylesheet" />
</head>

<body id="page-top">
    <!-- Navigation-->
    <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
        <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded mx-auto mb-2"
                src="assets/img/photo.jpg" alt="..." /></span>
        <a class="navbar-brand js-scroll-trigger" href="#page-top">
            <span class="d-block d-lg-none">Dominika Bohuslavová</span>
        </a>
        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive"
            aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span
                class="navbar-toggler-icon"></span></button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
            <ul class="navbar-nav">
                <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
                <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#education">Education</a></li>
                <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#projects">Projects</a></li>
                <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#activities">Activities</a></li>
                <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#publications">Publications</a></li>
                <!-- <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#interests">Interests</a></li> -->
                <!-- <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#awards">Awards</a></li> -->
            </ul>
        </div>
    </nav>
    <!-- Page Content-->
    <div class="container-fluid p-0">
        <!-- About-->
        <section class="resume-section" id="about">
            <div class="resume-section-content">
                <h1 class="mb-0">
                    Dominika
                    <span class="text-primary">Bohuslavová</span>
                </h1>
                <div class="subheading mb-5">
                    <a href="mailto:draesdom@fit.cvut.cz">draesdom@fit.cvut.cz</a>
                </div>
                <!-- <p class="lead mb-5">I know i know nothing</p> -->
                <div class="social-icons">
                    <a class="social-icon" href="https://www.linkedin.com/in/dominika-draesslerova/"><i
                            class="fab fa-linkedin-in"></i></a>
                    <a class="social-icon" href="https://github.com/draessld"><i class="fab fa-github"></i></a>
                    <a class="social-icon" href="https://x.com/draessld"><i class="fab fa-twitter"></i></a>
                    <a class="social-icon" href="https://orcid.org/0009-0006-6304-597X"><i class="fab fa-orcid"></i></a>
                    <!-- <a class="social-icon" href="https://dblp.org/search?q=Dominika+Draesslerova"><i class="fas fa-book"></i></a> -->
                    <!-- <a class="social-icon" href="#!"><i class="fab fa-facebook-f"></i></a> -->
                </div>
            </div>
        </section>
        <hr class="m-0" />
        <!-- Education-->
        <section class="resume-section" id="education">
            <div class="resume-section-content">
                <h2 class="mb-5">Education</h2>
                <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                    <div class="flex-grow-1">
                        <h3 class="mb-0">Theoretical Computer Science - Doctoral program</h3>
                        <div class="subheading mb-3">Czech Technical University in Prague (CTU)</div>
                        <!-- <p>describe</p> -->
                    </div>
                    <div class="flex-shrink-0"><span class="text-primary">February 2023 - Present</span></div>
                </div>
                <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                    <div class="flex-grow-1">
                        <h3 class="mb-0">Theoretical Computer Science - Master program</h3>
                        <div class="subheading mb-3">Czech Technical University in Prague (CTU)</div>
                        <!-- <p>Theoretical computer science will broaden your horizons in the theoretical foundations of computer science. You will deepen your knowledge in the areas of graph theory, string theory, data compression, arbology, compilers or (parameterized) complexity theory. In particular, you will learn about the design of solutions to hard problems and the development of algorithms for various branches of computer science. This specialization is also a great start for a scientific career, not only for CTU.</p> -->
                    </div>
                    <div class="flex-shrink-0"><span class="text-primary">September 2020 - February 2023</span></div>
                </div>
                <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                    <div class="flex-grow-1">
                        <h3 class="mb-0">Bioinformatics & chemical informatics - Master program</h3>
                        <div class="subheading mb-3">University of Chemical Technology, Prague (UCT)</div>
                        <!-- <p>You will be in a world that operates at the intersection of the natural sciences and computer science. Your daily bread will be chemoinformatics, bioinformatics or biomolecular modelling. You will become an expert with comprehensive knowledge of information technology, data analysis, chemistry, biochemistry and molecular biology. Crucially, you will learn to process, analyse and interpret experimental biological and chemical data and understand the intricate patterns of complex biological systems at the molecular level. You will be able to formulate research hypotheses, design procedures to test them and validate them using standard methods. You will be able to draw on a broad and versatile knowledge of computer science, and you will be an ideal link in interdisciplinary teams at the interface between the natural sciences and computer science. Graduates with this specialisation are in a minimum on the market, which is why they are perfectly applicable not only in the Czech Republic but all over the world.</p> -->
                        <p><b>Uncompleted</b></p>
                    </div>
                    <div class="flex-shrink-0"><span class="text-primary">September 2020 - January 2021</span></div>
                </div>
                <div class="d-flex flex-column flex-md-row justify-content-between">
                    <div class="flex-grow-1">
                        <h3 class="mb-0">Bioinformatics & chemical informatics - Bachelor program</h3>
                        <div class="subheading mb-3">University of Chemical Technology, Prague (UCT)</div>
                        <!-- <p>Bioinformatics is a scientific discipline that deals with computer methods for collecting, analyzing and visualizing large sets of biological data, especially data on the sequences and structures of nucleic acids or proteins. This degree programme therefore provides an education in chemical and molecular biological sciences combined with a balanced set of professional knowledge and skills in statistics, data analysis and computer science. Your broad outlook and interdisciplinary background will make you a valued specialist who will find employment in all sectors where biological sample processing and analysis occurs.</p> -->
                    </div>
                    <div class="flex-shrink-0"><span class="text-primary">September 2017 - June 2020</span></div>
                </div>
            </div>
        </section>
        <hr class="m-0" />
        <!-- Education-->
        <section class="resume-section" id="projects">
            <div class="resume-section-content">
                <h2 class="mb-5">Projects</h2>
                <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                    <div>I am working on my dissertation thesis on "Compressed indices for biological sequence
                        collections". The motivation behind this title is simple - to exploit the similarities between
                        sequences of the same or similar origin and to create a data structure that leads to efficient
                        querying over the data in terms of time and memory. My goal is to get my projects to a point
                        where they are well documented and potentially usable in bioinformatics practice.</div>
                </div>
                <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                    <div class="flex-grow-1">
                        <h3 class="mb-0">Data structures in taxonomic classification</h3>
                        <div></div>
                        <p>For taxonomic classification, we are asked to index the genomes in a phylogenetic tree such
                            that later, given a DNA read, we can quickly choose a small subtree likely to contain the
                            genome from which that read was drawn. Although popular classifiers such as Kraken use
                            k-mers, recent research indicates that using maximal exact matches (MEMs) can lead to better
                            classifications. For example, we can
                        <ul>
                            <li>

                                build an augmented FM-index over the the genomes in the tree concatenated in
                                left-to-right order;
                            </li>

                            <li>
                                for each MEM in a read, find the interval in the suffix array containing the starting
                                positions of that MEM’s occurrences in those genomes;
                            </li>

                            <li>
                                find the minimum and maximum values stored in that interval;
                            </li>
                            <li>

                                take the lowest common ancestor (LCA) of the genomes containing the characters at those
                                positions.
                            </li>
                        </ul>
                        This solution is practical, however, only when the total size of the genomes in the tree is
                        fairly small. In this paper we consider applying the same solution to three lossily compressed
                        representations of the genomes' concatenation:
                        <ul>
                            <li>
                                a KATKA kernel, which discards characters that are not in the first or last occurrence of any k_max-tuple, for a parameter k_max; 
                            </li>
                            <li>
                                a minimizer digest; 
                            </li>
                            <li>
                                a KATKA kernel of a minimizer digest. 
                            </li>
                        </ul> 
                        
                        With a test dataset and these three representations of it, simulated reads and various parameter settings, we checked how many reads' longest MEMs occurred only in the sequences from which those reads were generated ("true positive" reads). For some parameter settings we achieved significant compression while only slightly decreasing the true-positive rate. </p>
                    </div>
                </div>
                <div class="d-flex flex-column flex-md-row justify-content-between">
                    <div class="flex-grow-1">
                        <h3 class="mb-0">Elastic-degenerate strings</h3>
                        <div class="subheading mb-3"></div>
                        <p>
                            A genomic data can be stored in many different ways and formats, from simple sequence collection storing data directly in their raw form to variant calling formats (VCFs), which store more sophisticated information about individual variants. Most of these formats still compare new sequences to the reference sequence, limiting the analysis to a comparison to some consensus that has been established with limited information. This is not the only reason why EDS has found its place in pangenomics.
                        </p>
                        <p>
                            EDS is able to retain information from all the sequences in the collection with a reasonable reduction in memory consumption and thus remains representative of the species’ pangenome. Whether or not we keep information from the original collection (phasing of the sequences), we can predict variants haven’t yet been added to the collection.
                        </p>
                        <p>
                            The process of creating such an EDS is crucial, as there are many ways and parameters that strongly influence the information capacity, size and compression ratio of the structure.
                        </p>

                        <img id="pdfButton" src="assets/img/poster.png" alt="...">

                        <!-- <button id="pdfButton">Open PDF Overview</button> -->

                        <div class="overlay" id="overlay">
                            <button class="close-btn" id="closeBtn">Close</button>
                            <iframe id="pdfViewer" src="assets/img/ELIXIR_Annual_poster-1.pdf"></iframe>
                        </div>
                    </div>
                </div>
                <br>
                <br>
                <br>
                <div class="d-flex flex-column flex-md-row justify-content-between mb-5"></div>
                    <div class="flex-grow-1">
                        <h3 class="mb-0">Grammar indices</h3>
                        <div></div>
                        <p>In this project we focus on the operation, which can be done via highly developed grammar indices. But! This project is still quite new... do not hang up :)</p>
                    </div>
                </div>
            </div>
            </div>
        </section>
        <hr class="m-0" />
        <!-- Skills-->
        <section class="resume-section" id="activities">
            <div class="resume-section-content">
                <h2 class="mb-5">Activities</h2>
                <div class="subheading mb-3">2024</div>
                <ul class="list-inline">
                    <li><a href="https://psc.fit.cvut.cz/event/StringMasters2024/">Summer StringMasters 2024</a></li>
                    <li><a href="https://psc.fit.cvut.cz/event/2024/">Prague Stringology Conference </a></li>
                    <li><a href="https://sea2024.univie.ac.at/">Symposium on Experimental Algorithms (SEA) 2024 </a>
                    </li>
                    <li><a href="https://dsb-meeting.github.io/">Data Structures in Bioinformatics (DSB) 2024 </a></li>
                </ul>
                <div class="subheading mb-3">2023</div>
                <ul class="list-inline">
                    <li><a href="https://homepages.cwi.nl/~solon/alpan.html">ALPACA/PANGAIA ANNUAL WORKSHOP</a></li>
                    <li><a href="https://cpm2023.u-pem.fr/">34th Annual Symposium on Combinatorial Pattern Matching
                            (CPM) 2023</a></li>
                    <li><a href="https://sites.google.com/view/sequencesworkshop/home">Sequences in London</a></li>
                </ul>
                <h2 class="mb-5">Interns</h2>
                <div class="subheading mb-3">2024</div>
                <ul class="list-inline">
                    <li>April: <a href="https://www.cwi.nl/en/groups/networks-and-optimization/">CWI's Network and
                            Optimization research group</a></li>
                </ul>
                <div class="subheading mb-3">2022</div>
                <ul class="list-inline">
                    <li>July-November: <a href="https://ribbonbiolabs.com/">Ribbon Biolabs, Computational biologist
                            intern</a></li>
                </ul>
                <h2 class="mb-5">Teaching</h2>
                <ul class="list-inline">
                    <li>
                        <div class="subheading mb-3">Automata and Grammars tutorial (Theory of formal languages)</div>
                        <p>2021, 2023, 2024</p>
                    </li>
                </ul>

                <!-- <div class="subheading mb-3">Programming Languages & Tools</div>
                    <ul class="list-inline dev-icons">
                        <li class="list-inline-item"><i class="fab fa-html5"></i></li>
                        <li class="list-inline-item"><i class="fab fa-css3-alt"></i></li>
                        <li class="list-inline-item"><i class="fab fa-js-square"></i></li>
                        <li class="list-inline-item"><i class="fab fa-angular"></i></li>
                        <li class="list-inline-item"><i class="fab fa-react"></i></li>
                        <li class="list-inline-item"><i class="fab fa-node-js"></i></li>
                        <li class="list-inline-item"><i class="fab fa-sass"></i></li>
                        <li class="list-inline-item"><i class="fab fa-less"></i></li>
                        <li class="list-inline-item"><i class="fab fa-wordpress"></i></li>
                        <li class="list-inline-item"><i class="fab fa-gulp"></i></li>
                        <li class="list-inline-item"><i class="fab fa-grunt"></i></li>
                        <li class="list-inline-item"><i class="fab fa-npm"></i></li>
                    </ul> -->
                <!-- <div class="subheading mb-3">Workflow</div>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Mobile-First, Responsive Design
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Cross Browser Testing & Debugging
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Cross Functional Teams
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Agile Development & Scrum
                        </li>
                    </ul> -->
            </div>
        </section>
        <!-- <hr class="m-0" /> -->
        <!-- Interests-->
        <!-- <section class="resume-section" id="interests">
                <div class="resume-section-content">
                    <h2 class="mb-5">Interests</h2>
                    <p>Apart from being a web developer, I enjoy most of my time being outdoors. In the winter, I am an avid skier and novice ice climber. During the warmer months here in Colorado, I enjoy mountain biking, free climbing, and kayaking.</p>
                    <p class="mb-0">When forced indoors, I follow a number of sci-fi and fantasy genre movies and television shows, I am an aspiring chef, and I spend a large amount of my free time exploring the latest technology advancements in the front-end web development world.</p>
                </div>
            </section> -->
        <!-- <hr class="m-0" /> -->
        <!-- Awards-->
        <section class="resume-section" id="publications">
            <div class="resume-section-content">
                <h2 class="mb-5">Publications</h2>
                <div class="subheading mb-3">2024</div>
                <ul class="fa-ul mb-0">
                    <li>
                        <span class="fa-li"><i class="fa-solid fa-bars"></i></span>
                        Dominika Draesslerová, Omar Ahmed, Travis Gagie, Jan Holub, Ben Langmead, Giovanni Manzini, and
                        Gonzalo Navarro. Taxonomic Classification with Maximal Exact Matches in KATKA Kernels and
                        Minimizer Digests. In 22nd International Symposium on Experimental Algorithms (SEA 2024).
                        Leibniz International Proceedings in Informatics (LIPIcs), Volume 301, pp. 10:1-10:13, Schloss
                        Dagstuhl – Leibniz-Zentrum für Informatik (2024)
                        <a
                            href="https://doi.org/10.4230/LIPIcs.SEA.2024.10">https://doi.org/10.4230/LIPIcs.SEA.2024.10</a>
                    </li>
                </ul>
            </div>
        </section>
    </div>
    <!-- Bootstrap core JS-->
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
    <!-- Core theme JS-->
    <script src="js/scripts.js"></script>
</body>

</html>